website/docs/getting-started/getting-started-guide.md

```diff
@@ -40,10 +40,9 @@ Use "ksctl [command] --help" for more information about a command.
 ```
 
 ## Authenticate with cloud provider
-First, you will need to authenticate with your chosen cloud provider. Here we are considering Civo cloud provider
+First, you will need to authenticate with your chosen cloud provider.
 
-![](https://i.imgur.com/jexwOeu.png)
-Copy the token and then Open a terminal and type in `ksctl cred`
+Once you have obtained the authentication token or credentials, open a terminal and run the following command:
```

Comment:
below this line provide the links for obtaining the credenaitls from the providers section